src/app/components/client/Chart.tsx

```diff
@@ -90,12 +91,19 @@ export const DoughnutChart = (props: Props) => {
   const modalContent = (
     <div className={styles.modalBodyContent}>
       <p>
-        {l10n.getString("modal-active-number-of-exposures-part-one", {
-          limit: 5,
-        })}
+        {l10n.getString(
+          props.isEligibleForPremium
+            ? "modal-active-number-of-exposures-part-one-premium"
+            : "modal-active-number-of-exposures-part-one-all",
+          {
+            limit: 5,
```

Comment:
And actually that env var aleady existed without `NEXT_PUBLIC`. I've just added it for now, since I do expect that change to get merged in in the short term: 5a7766d60.

Reply:
Oh heh, of course the tests then fail. But I can just make the same change here and Git can reconcile it whenever the second PR gets merged: d6b39de0e